homeassistant/components/mqtt/vacuum/schema_state.py

```diff
@@ -229,14 +253,15 @@ def state_message_received(msg):
             self.hass, self._sub_state, topics
         )
 
-    async def _subscribe_topics(self):
+    async def _subscribe_topics(self) -> None:
         """(Re)Subscribe to topics."""
         await subscription.async_subscribe_topics(self.hass, self._sub_state)
 
-    async def async_start(self):
+    async def async_start(self) -> None:
         """Start the vacuum."""
         if self.supported_features & VacuumEntityFeature.START == 0:
             return None
+        assert self._command_topic is not None
```

Comment:
As above, I think these asserts should be removed, replaced by early exits inside the supported_features check.